doc/task_breakdown.md

```diff
@@ -0,0 +1,99 @@
+# SQL Query Builder: Task Breakdown
+
+This is a breakdown of tasks for building the **SQL Query Builder** application.
+
+---
+
+## 1. Interface for Building SQL Queries
```

Comment:
You mean that this is one big task? just to create UI? 

Or each sub- item for this is also a task?